src/components/ProductTable/index.tsx

```diff
@@ -6,41 +6,174 @@ import Image from "next/image";
 import { toast } from "sonner";
 import { LuClipboardEdit } from "react-icons/lu";
 import { LuTrash2 } from "react-icons/lu";
+import { LuArrowDownZA } from "react-icons/lu"; // z - a
+import { LuArrowDownAZ } from "react-icons/lu"; // a - z
+
+import { LuArrowDown01 } from "react-icons/lu"; // 0 - 1
+import { LuArrowDown10 } from "react-icons/lu"; // 1 - 0
 
 export interface ProductTableState {
   isLoading: boolean;
   page: number;
   totalPages: undefined | number;
   currentProducts: Product[];
   limit: number;
+  sort: {
+    prop: string;
+    order: "asc" | "desc";
+    filter: null | string;
+  };
 }
 
+const thArr: {
```

Comment:
Maybe we can find a better name for this, something that explains what is the content.

Reply:
Check the new one. If it doesn't suit with your name convention, you can suggest one